packages/admin/admin-generator/src/commands/generate/generateGrid/generateGqlFieldList.ts

```diff
@@ -22,7 +22,13 @@ const recursiveStringify = (obj: FieldsObjectType): string => {
 
 export function generateGqlFieldList<T extends { __typename?: string }>({ columns }: { columns: GridConfig<T>["columns"] }) {
     const fieldsObject: FieldsObjectType = columns.reduce<FieldsObjectType>((acc, field) => {
-        if (field.type !== "actions") {
+        if (field.type === "actions") {
+            if ("queryFields" in field) {
+                field.queryFields?.forEach((queryField) => {
+                    objectPath.set(acc, queryField, true);
+                });
+            }
```

Comment:
do you really theed the if?
```suggestion
            field.queryFields?.forEach((queryField) => {
                objectPath.set(acc, queryField, true);
            });
```